lib/matplotlib/backends/backend_qt5.py

```diff
@@ -59,7 +59,7 @@
                 QtCore.Qt.Key_Clear: 'clear', }
 
 # define which modifier keys are collected on keyboard events.
-# elements are (mpl names, Modifier Flag, Qt Key) tuples
+# elements are (Matplotlib names, Modifier Flag, Qt Key) tuples
```

Comment:
Maybe better:
```suggestion
# elements are (modifier name, Modifier Flag, Qt Key) tuples
```

or `matplotlib modifier name` if you want to be really precise.